.github/workflows/_featurenet-create.yml

```diff
@@ -40,6 +40,11 @@ on:
         required: false
         type: boolean
         default: false
+      sudo-account-id:
+        description: 'Sudo account ID'
+        type: string
+        required: true
```

Comment:
Why is there `required: true` when it's got a default value?

Reply:
good point, it should be `false` since otherwise it's a API breaking change